web/rocket/2_json_apis/src/main.rs

```diff
@@ -0,0 +1,57 @@
+#[macro_use]
+extern crate rocket;
```

Comment:
Why we are using 'extern crate' here??

Reply:
it's just a shorthand, I skip importing multiple components one by one instead I did it all at once using `extern crate`.

Here are some additional things that I had to do if I don't use `extern crate`.
```suggestion
use rocket::{get, launch, routes, Build, Rocket};
```

```suggestion
#[launch]
fn rocket() -> Rocket<Build> {
    // mounting various routes in the rocket server
    rocket::build().mount("/", routes![hello, get_json, add_user])
}
```